Scenario/Plugins/PluginSettings.cs

```diff
@@ -16,8 +16,7 @@ public class PluginSettings : EditorWindow
 
         private readonly string[] imageFormats = { "JPEG", "PNG" };
         private readonly string[] imageFormatExtensions = { "jpeg", "png" };
-
-        private static string version = "Scenario Beta Version 0.2.0";
+        private static string version = "Scenario Beta Version 0.2.3";
```

Comment:
Is there any possibility to use the version number in package.json rather than hardcoding it?